es6/README.md

```diff
@@ -0,0 +1,403 @@
+# Linting
+
+Included is a default ESLint configuration, `mobify-es6.yml` written for ESLint 3.x. It requires the additional modules:
+
+- `eslint-plugin-import`
+- `eslint-import-resolver-webpack`
```

Comment:
We could do that, though it would also install them for ES5 projects that use the `mobify-code-style` module.